modules/osdk-updating-128-to-131.adoc

```diff
@@ -0,0 +1,221 @@
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver-n1}` to `{osdk_ver}`.
+
+. If you use a custom service account for deployment, define the following role to require a watch operation on your secrets resource, as shown in the following example:
++
+.Example `config/rbac/role.yaml` file
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <operator_name>-admin
+subjects:
+- kind: ServiceAccount
+  name: <operator_name>
+  namespace: <operator_namespace>
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+rules: <1>
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - watch
+----
+<1> Add the `rules` stanza to create a watch operation for your secrets resource.
+endif::[]
+
+ifdef::ansible[]
+
+. Make the following changes to your Operator's Dockerfile:
+
+.. Replace the `ansible-operator-2.11-preview` base image with the `ansible-operator` base image and update the version to {osdk_ver}, as shown in the following example:
++
+.Example Dockerfile
+[source,docker,subs="attributes+"]
+----
+FROM quay.io/operator-framework/ansible-operator:v{osdk_ver}
+----
+
+.. The update to Ansible 2.15.0 in version 1.30.0 of the Ansible Operator removed the following pre-installed Python modules:
++
+--
+* `ipaddress`
+* `openshift`
+* `jmespath`
+* `cryptography`
+* `oauthlib`
+--
++
+If your Operator depends on one of these removed Python modules, update your Dockerfile to install the required modules using the `pip install` command.
+
+. Edit your Operator project's makefile to update the Operator SDK version to {osdk_ver}, as shown in the following example:
++
+.Example makefile
+[source,make,subs="attributes+"]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver-n1}` to `{osdk_ver}`.
+
+. Update your `requirements.yaml` and `requirements.go` files to remove the `community.kubernetes` collection and update the `operator_sdk.util` collection to version `0.5.0`, as shown in the following example:
++
+.Example `requirements.yaml` file
+[source,diff]
+----
+  collections:
+-  - name: community.kubernetes <1>
+-    version: "2.0.1"
+   - name: operator_sdk.util
+-    version: "0.4.0"
++    version: "0.5.0" <2>
+   - name: kubernetes.core
+     version: "2.4.0"
+   - name: cloud.common
+----
+<1> Remove the `community.kubernetes` collection
+<2> Update the `operator_sdk.util` collection to version `0.5.0`.
+
+. Remove all instances of the `lint` field from your `molecule/kind/molecule.yml` and `molecule/default/molecule.yml` files, as shown in the following example:
```

Comment:
Despite the fact that this step looks off in the files, it renders OK. Not sure why its gray here.